src/server/game/Handlers/MiscHandler.cpp

```diff
@@ -1062,11 +1062,17 @@ void WorldSession::HandleSetActionButtonOpcode(WorldPacket& recvData)
 
 void WorldSession::HandleCompleteCinematic(WorldPacket& /*recvData*/)
 {
+    // If player has sight bound to visual waypoint NPC we should remove it
+    if (Player* thisPlayer = GetPlayer())
```

Comment:
same here, GetPlayer() never be null

while you are on these lines, pls remove the useless network log messages